### PR TITLE
fix: top loader bug

### PR DESCRIPTION
**Pull request by tomcru**
## Describe your changes

closes https://github.com/BearStudio/start-ui-web/issues/434

## Checklist

 - [x] I performed a self review of my code
 - [x] I ensured that everything is written in English
 - [x] I tested the feature or fix on my local environment
 - [x] I ran the `pnpm storybook` command and everything is working
<s> - [ ] If applicable, I updated the translations for english and french files  
      (If you cannot update the french language, just let us know in the PR description)</s>
<s>- [ ] If applicable, I updated the README.md</s>
<s> - [ ] If applicable, I created a PR or an issue on the [documentation repository](https://github.com/bearstudio/start-ui-web-docs/)</s>
 - [x] If applicable, I’m sure that my feature or my component is mobile first and available correctly on desktop

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,11 +67,11 @@
     "colorette": "2.0.20",
     "dayjs": "1.11.10",
     "framer-motion": "10.16.15",
+    "holy-loader": "^2.2.5",
```
Comment (ivan-dalmet) [inline package.json:70]: Please remove the `^` to make the dependency stable?

**Review by ivan-dalmet — CHANGES_REQUESTED** [review]
Thank for the PR 👍
I'm on mobile right now looking at the preview deployment but it looks like that the showSpinner false in not working on the admin part.

(also if you can fix the dependency version 👌)

**Comment by tomcru** [comment]
`showSpinner={false}` is properly working on the admin pages now and added `^` to make the dependency stable

**Review by ivan-dalmet — APPROVED** [review]

**Comment by ivan-dalmet** [comment]
Thanks a lot @tomcru ❤️

**Pull request merged**